### ISSUE TITLE
failed to resolve component vue3-simple-typeahead

### ISSUE THREAD

**Issue by bci24** [frikinside/vue3-simple-typeahead#12]
The component is not working on vue 3 (composition api)

I need this in a component not globally

In my custom component I imported like in the example:

```
import SimpleTypeahead from 'vue3-simple-typeahead''
export default {
  name: 'my-vue-component',
  ...
  components: {
    SimpleTypeahead
  }
  ...
}
```

but it does not appear and I get a warning:

```
Failed to resolve component: vue3-simple-typeahead
If this is a native custom element, make sure to exclude it from component resolution via compiler.isCustomElement .......
```

Any idea?

**Comment by frikinside** [comment]
Sorry that keep you waiting, I did a few tests, but could reproduce the problem.

I see no problem using the component with composition api, do you still have this problem?

P.D. Yes, I just updated the component adding fallingthrought attributes so you can add whatever attribute you want (like class) and it fallthrough to the input.

**Comment by BradA1878** [comment]
@bci24 & @frikinside - This is a TypeScript error due to there not being a type declaration for the component. You can get around the issue by adding `/ // @ts-ignore ` above the line where the import is made.

**Comment by frikinside** [comment]
Thanks for the input @BradA1878!!
@bci24 could you please try the solution provided by @BradA1878 and check if that solve your issue?

**Comment by x4m3** [comment]
I also had this typescript error, when I was building my project (with `vite`) I got this issue:

<img width="1680" alt="Screenshot 2022-08-22 at 01 04 45" src="https://user-images.githubusercontent.com/8809909/185814723-42bd8862-ae84-413c-8c58-7f0f0a881d3c.png">

I fixed the issue by creating the file `vue3-simple-typeahead.d.ts` anywhere in my project with the following content (like the error message is saying):
```typescript
declare module "vue3-simple-typeahead";
```

Yes, there's no typescript types, but at least the build error is gone :)

**Comment by frikinside** [comment]
Thanks a lot @x4m3 I'm not used to typescript so I find this kind of comments incredible valuable!
Very nice way to "trick" typescript to avoid the build error.

There is no detrimental problems with that, right?

**Comment by x4m3** [comment]
no, there's no detrimental problems, there just won't be any typescript typings, but it's still going to be valid javascript
